packages/opentelemetry-exporter-jaeger/src/transform.ts

```diff
@@ -88,18 +88,16 @@ export function spanToThrift(span: ReadableSpan): ThriftSpan {
 
   const spanTags: ThriftTag[] = ThriftUtils.getThriftTags(tags);
 
-  const logs = span.events.map(
-    (event): Log => {
-      const fields: Tag[] = [{ key: 'event', value: event.name }];
-      const attrs = event.attributes;
-      if (attrs) {
-        Object.keys(attrs).forEach(attr =>
-          fields.push({ key: attr, value: toTagValue(attrs[attr]) })
-        );
-      }
-      return { timestamp: hrTimeToMilliseconds(event.time), fields };
+  const logs = span.events.map((event): Log => {
+    const fields: Tag[] = [{ key: 'event', value: event.name }];
+    const attrs = event.attributes;
+    if (attrs) {
+      Object.keys(attrs).forEach(attr =>
+        fields.push({ key: attr, value: toTagValue(attrs[attr]) })
+      );
     }
-  );
+    return { timestamp: hrTimeToMilliseconds(event.time), fields };
+  });
```

Comment:
I am not sure why running `lint:fix` changed this code which has no modifications.
Any ideas?

Reply:
Probably just an updated linter since it was last fixed